static/app/views/replays/detail/ai/ai.tsx

```diff
@@ -91,7 +93,8 @@ export default function Ai() {
 
   // If our `replay-ai-summaries` ff is enabled and the org has gen AI ff enabled,
   // but the org hasn't acknowledged the gen AI features, then show CTA.
-  if (!setupAcknowledgement.orgHasAcknowledged) {
+  // only do this if consent flow is not skipped
+  if (!skipConsentFlow && !setupAcknowledgement.orgHasAcknowledged) {
```

Comment:
Technically this isn't necessary since the flag will make `orgHasAcknowledged` be true